imu_preprocess_node:
  ros__parameters:
    # Pre-calibrated IMU bias values
    use_precalibrated_bias: true
    
    # Accelerometer bias [m/sÂ²]
    bias_acc_x: -0.20183853281631028
    bias_acc_y: 0.026593970509083378
    bias_acc_z: -0.23629274367340436
    
    # Gyroscope bias [rad/s]
    bias_gyro_x: 0.0019944836239094523
    bias_gyro_y: -0.014121080980058574
    bias_gyro_z: -0.02373539930695253
    
    # Low-pass filter settings
    lpf_cutoff: 5.0  # 0 to disable LPF